### DISCUSSION TITLE
TypeError: ctx.params.pipe is not a function

### DISCUSSION THREAD

**Discussion by putty-kang** [moleculerjs/moleculer#1233] [Q&A]
![image](https://github.com/moleculerjs/moleculer/assets/54395226/6fe2571a-ae24-4ba9-954c-78b48061fb10)

May I ask how to handle this

**Comment by icebob** [comment]
The `pipe` is a method of a Stream, the error means, you didn't call the action with a `Stream` instance. The `encryptedData` is a `String`, and no `streaming: true` calling option.

Please read tha manual, how streaming is work: https://moleculer.services/docs/0.14/actions#Streaming

**Reply by putty-kang** [reply]
I want to stream a string, how do I handle this

**Reply by icebob** [reply]
You can find some solution on the Internet: https://stackoverflow.com/questions/12755997/how-to-create-streams-from-string-in-node-js
